### ISSUE TITLE
Sysmon is a legacy name title and contents should be updated to Performance Monitor

### ISSUE THREAD

**Issue by jasonhorner** [MicrosoftDocs/sql-docs#6860]
Sysmon was the older name for Performance Monitor

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: acc780ea-498b-15ac-1c05-2251d8dde598
* Version Independent ID: e5c5dc30-2920-9887-3062-969e5245d2ac
* Content: [Monitoring Replication with System Monitor - SQL Server](https://docs.microsoft.com/en-us/sql/relational-databases/replication/monitor/monitoring-replication-with-system-monitor?view=sql-server-ver15#feedback)
* Content Source: [docs/relational-databases/replication/monitor/monitoring-replication-with-system-monitor.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/relational-databases/replication/monitor/monitoring-replication-with-system-monitor.md)
* Product: **sql**
* Technology: **replication**
* GitHub Login: @MashaMSFT
* Microsoft Alias: **mathoma**

**Comment by WilliamAntonRohm** [comment]
@jasonhorner -- thank you for your feedback.  I am updating the article.

**Comment by WilliamDAssafMSFT** [comment]
@WilliamAntonRohm This is correct and needs to be updated in several places. Happy to review. We also say "System Monitor replaced Performance Monitor after Windows NT 4.0." in a few docs. This was true back then but Performance Monitor has since replaced System Monitor. We can remove notes pertaining to System Monitor or Windows NT 4.0. I believe in 2003 System Monitor was within the Performance Monitor MMC... but certainly on modern supported OSs, Performance Monitor is the name of the tool. Certainly the capturing of WMI performance objects these days with a built-in windows server OS tool is done with Performance Monitor.
Not to be confused with [Sysmon](https://docs.microsoft.com/sysinternals/downloads/sysmon) which is a modern sysinternals tool.
